en/includes/guides/authorization/user-impersonation/user-impersonation-via-business-app.md

```diff
@@ -274,7 +273,6 @@ The decoded access token looks as follows:
 
 - The `act.sub` property holds the user ID of the impersonator. In the given exmaple, the `act.sub` property holds the value of `2d931c9d-876e-46c0-9aba-f34501879dfc`, the user ID of the impersonator.
 
- 
 ## Configure more impersonation options
 
-To learn how to check impersonation logs, notify users on impersonation and how to access child organizations as an impersonator, see the end of the [User impersonation via Console]({{base_path}}/guides/authorization/user-impersonation/via-console/) guide.
\ No newline at end of file
+To learn how to check impersonation logs, notify users on impersonation and how to access child organizations as an impersonator, see the end of the [User impersonation via Console]({{base_path}}/guides/authorization/user-impersonation/via-console/) guide.
```

Comment:
What is the change we did here?

Reply:
No difference, may be a empty character might have caused the change.